adafruit_platformdetect/board.py

```diff
@@ -43,6 +44,8 @@
 ODROID_C1                   = "ODROID_C1"
 ODROID_C1_PLUS              = "ODROID_C1_PLUS"
 ODROID_C2                   = "ODROID_C2"
+
+FTDI_FT232H                 = "FT232H"
```

Comment:
just so i'm clear on this, the FT232H is a situation where we're using a peripheral on a regular desktop system to provide interfaces, rather than a separate platform we're running python on directly?

it seems like this might be jamming two concepts together under the same abstraction, but maybe it doesn't matter...

Reply:
yes,  you will be able to use the FT232H as a circuitpython-library-interface and use that for interfacing hardware. i could shove this into blinka but it would make the code a lot more complex because i'd have to do so many tests. i think...this is OK. you do need to set up an envvar, its not like it will happen by accident!

Reply:
cool, makes sense.